services/QuillLMS/client/app/bundles/Grammar/actions/session.ts

```diff
@@ -126,11 +127,12 @@ const normalizeQuestionWithAttempts = (question: object): object => {
     attempts: question.attempts,
   }
 }
+const activityUID = getParameterByName('uid', window.location.href)
```

Comment:
I'm not totally sure when this gets instantiated, or if it'll change when the URL changes. It might be safest to make it into a function (ex `const activityUID = () => getParameterByName...`) so you can be confident it's getting defined when it's called and not holding onto something stale.

Reply:
yep, it causes problems in admin if you don't make it a function, good catch.